transportbase.go

```diff
@@ -111,7 +113,7 @@ func (b *TransportBase) acceptStreams() {
 	for {
 		s, err := b.session.AcceptStream()
 		if err != nil {
-			b.log.Errorf("Failed to accept stream: %v", err)
+			fmt.Fprintf(os.Stderr, "Failed to accept stream: %T %v\n", err, err)
```

Comment:
I would use `b.log.Errorf` if possible. Users can control this with by setting env variables [like this](https://github.com/pion/webrtc/wiki/Debugging-WebRTC) OR they can redirect to a custom log file like [this](https://github.com/pion/webrtc/tree/master/examples/custom-logger)

Reply:
Thanks! I was looking for that and will change it back.

The testsuite setup somehow didn't set it which lead to a panic.